### PR TITLE
describe the location of rpmlintrc

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,8 @@ Additional option to control `rpmlint` behaviour is the addition of `rpmlintrc`
 which uses old syntax for compatibility with old `rpmlint` releases, yet
 it can be normal `toml` file if you wish:
 
-`setBadness('check', 0)`
-
-`addFilter('test-i-ignore')`
```
Comment (marxin) [inline README.md:127]: Please preserve the monospace for these lines.

**Review by marxin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xsuchy) [reply README.md:127]: It is preserved via 4 space indentation (which is IMO more readble). See the preview here https://github.com/rpm-software-management/rpmlint/blob/a8660e981299588417c3d27afd84ec6923e19c86/README.md

Comment (marxin) [reply README.md:127]: All right. So please rebase the pull request, we have conflicts now.

**Comment by xsuchy** [comment]
rebased

**Pull request merged**